src/App.tsx

```diff
@@ -1,61 +1,184 @@
+import { useState } from 'react';
 import './App.scss';
+import users from './api/users';
+import todos from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialUser: User = {
+  id: 0,
+  name: '',
+  username: '',
+  email: '',
+};
+const initialTodo: Todo = {
+  id: 0,
+  title: '',
+  completed: false,
+  userId: 0,
+  user: initialUser,
+};
+const findUser = (id: number): User => (
+  users.filter(user => user.id === id)[0]
```

Comment:
better to use the find method, 'cause you want to find the user

Reply:
Done!